src/main/scala/com/campudus/tableaux/Starter.scala

```diff
@@ -51,12 +51,10 @@ class Starter extends ScalaVerticle {
       val workingDirectory = getStringDefault(config, "workingDirectory", Starter.DEFAULT_WORKING_DIRECTORY)
       val uploadsDirectory = getStringDefault(config, "uploadsDirectory", Starter.DEFAULT_UPLOADS_DIRECTORY)
 
-      val tableauxConfig = TableauxConfig(
-        verticle = this,
-        databaseConfig = databaseConfig,
-        workingDir = workingDirectory,
-        uploadsDir = uploadsDirectory
-      )
+      val tableauxConfig = TableauxConfig(verticle = this,
+                                          databaseConfig = databaseConfig,
+                                          workingDir = workingDirectory,
+                                          uploadsDir = uploadsDirectory)
```

Comment:
Can we change that? As Scala lines tend to get quite long, I'd rather have it like it was before:
```scala
val tableauxConfig = new TableauxConfig(
  here = follows,
  the = configuration,
  options = with,
  just = two,
  spaces = inFront
)
```